demo/cca/manual-end-to-end.md

```diff
@@ -0,0 +1,295 @@
+created "comid-cca-mult-refval.cbor" from "data/templates/comid-cca-mult-refval.json"
+
+### Create CoRIM
+
+* Create a single CoRIM using given CoRIM JSON template and supplying CBOR Trust Anchor and Reference Value CoMIDs from above
+
+```shell
+cocli corim create --template=data/templates/corim-cca.json --comid=comid-cca-ta-endorsements.cbor --comid=comid-cca-mult-refval.cbor
+```
+created "corim-cca.cbor" from "data/templates/corim-cca.json"
+
+### Using newly generated CoRIM
+Move the generated CORIM above to data/cbor directory
+ 
+```shell
+mv corim-cca.cbor  data/cbor/
+```
+* Please retain this shell, as shell-1 as you would need to come back here at the time of provisioning the endorsements.
+
+## Provisioning pipeline setup
+
+To run provisioning test you need two parallel shell sessions.
+
+In each shell, move to the directory location where you are going to clone the GIT repo and do:
+
+```shell
+export TOPDIR=$(pwd)
+```
+
+### Steps
+
+In the first shell, clone the Veraison services repository:
+
+```shell
+git clone https://github.com/veraison/services@demo-cca-1.0.0
+```
+
+Build the services:
+
+```shell
+make -C ${TOPDIR}/services
+```
+
+Start the REST API frontend:
+
+```shell
+( cd ${TOPDIR}/services/provisioning/cmd/provisioning-service && ./provisioning-service )
+```
+
+In another shell create the KV stores:
+
+```shell
+( cd ${TOPDIR}/services/vts/cmd/vts-service && ./../../test-harness/init-kvstores.sh )
+````
+
+Then start the VTS service:
+
+```shell
+( cd ${TOPDIR}/services/vts/cmd/vts-service && ./vts-service )
+```
+VTS Service starts all the supported plugins (scheme-psa-iot, scheme-tcg-dice, scheme-tpm-enacttrust, scheme-cca-ssd-platform for now)
+
+Go back to shell-1, used for Creation of CCA Endorsements
+
+Ensure that you are under docs/demo/cca/prov-verif-e2e directory:
+
+```shell
+cd ${TOPDIR}/docs/demo/cca/prov-verif-e2e
+```
+
+Ship the single CORIM that contains CCA reference values and trust anchor from data/cbor folder using following commands
+
+
+```shell
+cocli corim submit --corim-file=data/cbor/corim-cca.cbor --api-server="http://localhost:8888/endorsement-provisioning/v1/submit" --media-type="application/corim-unsigned+cbor; profile=http://arm.com/cca/ssd/1"
+```
+
+The REST frontend should return a success status.
+
+The success of post API status can be noticed in console logs of front end (provisioning-service) likewise below...
+
+`[GIN] 2022/09/09 - 17:39:55 | 200 | 49.977785ms | 127.0.0.1 | POST "/endorsement-provisioning/v1/submit"`
+
+If so, you can inspect the KV stores to check what has been generated:
+
+* Verification keys:
+
+```shell
+sqlite3 ${TOPDIR}/services/vts/cmd/vts-service/ta-store.sql 'select distinct vals from kvstore' | jq .
+```
+
+* Reference values:
+
+```shell
+sqlite3 ${TOPDIR}/services/vts/cmd/vts-service/en-store.sql 'select distinct vals from kvstore' | jq .
+```
+
+### Cleanup
+
+Use Ctrl-C to stop the REST provisioning frontend in shells 1 (as one
```

Comment:
```suggestion
Use `Ctrl-C` to stop the REST provisioning frontend in shells 1 (as you
```

Reply:
Since you're goving direct instruction to the user there ("Use CTRL-C to..."), in the parens, you should use the second person pronoun, rather than the passive "one").